src/main/java/org/opentripplanner/middleware/docs/ReadMeEnvSchemaValuesUpdater.java

```diff
@@ -0,0 +1,103 @@
+     */
+    public static String generateEnvSchemaValuesContent() throws IOException {
+        FileInputStream envSchemaStream = new FileInputStream(ConfigUtils.DEFAULT_ENV_SCHEMA);
+        JsonNode envSchema = yamlMapper.readTree(envSchemaStream);
+        if (envSchema == null) {
+            throw new IllegalArgumentException("env.schema.json not available to update README.md.");
+        }
+        StringBuilder tableData = new StringBuilder("| Key | Type | Required | Default | Description |");
+        tableData.append(System.lineSeparator())
+                .append("| --- | --- | --- | --- | --- |")
+                .append(System.lineSeparator());
+        Iterator<String> propertyNames = envSchema.get("properties").fieldNames();
+        List<String> requiredProperties = (envSchema.get("required") != null)
+                ? getPOJOFromJSONAsList(envSchema.get("required"), String.class)
+                : Collections.emptyList();
+        JsonNode properties = envSchema.get("properties");
+        while (propertyNames.hasNext()) {
+            String propertyName = propertyNames.next();
+            JsonNode property = properties.get(propertyName);
+            String type = (property.get("type") != null)
+                    ? property.get("type").toString()
+                    : "";
+            String required = (requiredProperties
+                    .stream()
+                    .anyMatch(propertyName::equals))
+                    ? "Required"
+                    : "Optional";
+            String defaultValue = (property.get("default") != null)
+                    ? property.get("default").toString()
+                    : "";
+            String description = (property.get("description") != null)
+                    ? property.get("description").toString()
+                    : "";
+            tableData.append("| ")
+                    .append(propertyName)
+                    .append(" | ")
+                    .append(type.replaceAll("\"", ""))
+                    .append(" | ")
+                    .append(required)
+                    .append(" | ")
+                    .append(defaultValue.replaceAll("\"", ""))
+                    .append(" | ")
+                    .append(description.replaceAll("\"", ""))
+                    .append(" |")
+                    .append(System.lineSeparator());
+        }
+        return tableData.toString();
+    }
+
+    /**
+     * Updates the {@link #LATEST_README_FILE} file with the latest env.schema.json values. It is expected that the
+     * title for this section be named '### env.schema.json values' and be the last section in the README.md file. Any
+     * sections after this will be removed as part of this update process!
+     */
+    private static void updateEnvSchemaValues() throws IOException {
+        Path readMeFile = new File(LATEST_README_FILE).toPath();
+        String readMeContent = new String(Files.readAllBytes(readMeFile));
```

Comment:
`FileUtils#getFileContents`

Reply:
Updated with `Files.readString` to match unit test.